python/GafferUI/MenuBar.py

```diff
@@ -61,6 +63,15 @@ def __init__( self, definition, **kw ) :
 		self.parentChangedSignal().connect( Gaffer.WeakMethod( self.__parentChanged ), scoped = False )
 		self.__setupShortcutEventFilter()
 
+	## Adds a listener to the supplied gaffer widget to action any menu items
+	# in response to keyboard shortcut events received by that widget.
+	# This can be useful to ensure secondary windows not in the hierarchy of
+	# the MenuBar's parent still cause associated actions to fire.
+	def addShortcutTarget( self, widget ) :
+
+		widget._shortcutEventFilter = _ShortcutEventFilter( widget._qtWidget(), self )
```

Comment:
Use `widget.__shortcutEventFilter` (double underscore prefix) to avoid potential clashes with attributes managed by other classes.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3359/commits/df29839b9474c144393d20ffb51ac3af6ade514e, sorry was a hang over from experimental times.